packages/utils/src/lib/reports/formatting.unit.test.ts

```diff
@@ -76,51 +64,51 @@ describe('tableSection', () => {
           ],
         },
         { level: 3 },
-      ),
+      )?.toString(),
     ).toMatchSnapshot();
   });
 });
 
 describe('metaDescription', () => {
   it('should return empty string if no options are given', () => {
-    expect(metaDescription({})).toBe('');
+    expect(metaDescription({}).toString()).toBe('');
   });
 
   it('should return description if only description is given', () => {
     expect(
       metaDescription({
         description: 'Audit to track bundle size',
-      }),
-    ).toBe(`Audit to track bundle size${NEW_LINE}`);
+      }).toString(),
+    ).toBe('Audit to track bundle size');
   });
 
   it('should return docsUrl if only docsUrl is given', () => {
     expect(
       metaDescription({
-        docsUrl: `http://code-pushup.dev/audits/#lcp`,
-      }),
-    ).toBe(`[📖 Docs](http://code-pushup.dev/audits/#lcp)${NEW_LINE}`);
+        docsUrl: 'http://code-pushup.dev/audits/#lcp',
+      }).toString(),
+    ).toBe('[📖 Docs](http://code-pushup.dev/audits/#lcp)');
   });
 
   it('should docs and description if both given', () => {
     expect(
       metaDescription({
         description: 'Audit for loading performance',
         docsUrl: 'http://code-pushup.dev/audits/#lcp',
-      }),
+      }).toString(),
     ).toBe(
-      `Audit for loading performance [📖 Docs](http://code-pushup.dev/audits/#lcp)${NEW_LINE}`,
```

Comment:
I assume now it takes care of new lines automatically 🤔 ?

Reply:
Yes, new lines are now handled automatically by the `build-md` library when the final document is rendered. This function only returns the inner contents of the paragraph, not the paragraph block (that's taken care of in the caller function), so no block formatting is applied yet.